src/ui/components/Scanner/Scanner.tsx

```diff
@@ -162,18 +181,13 @@ const Scanner = forwardRef(
           dispatch(setToastMsg(ToastMsgType.NEW_MULTI_SIGN_MEMBER));
         }
       } catch (e) {
-        // eslint-disable-next-line no-console
-        console.error("Scanner Error:", e);
-        dispatch(setToastMsg(ToastMsgType.SCANNER_ERROR));
+        showError("Scanner Error:", e, dispatch, ToastMsgType.SCANNER_ERROR);
         initScan();
       }
     };
 
     const processValue = async (content: string) => {
       stopScan();
-      // @TODO - foconnor: instead of setting the optype to idle we should
-      // have a loading screen with "waiting for server..." etc,
-      // and it can update to an error if the QR is invalid with a re-scan btn
```

Comment:
Why are we deleting these comments? I think we can leave this and see if we can create a ticket for it.

Reply:
Done. I revert it.